src/lib/import_export/csv_parser.cpp

```diff
@@ -67,56 +77,17 @@ std::shared_ptr<Table> CsvParser::parse(const std::string& filename) {
   return table;
 }
 
-std::shared_ptr<Table> CsvParser::_process_meta_file(const std::string& filename) {
-  const char delimiter = '\n';
-  const char separator = ',';
-
-  std::ifstream metafile{filename};
-  std::string content{std::istreambuf_iterator<char>(metafile), {}};
-
-  // make sure content ends with '\n' for better row processing later
-  if (content.back() != delimiter) content.push_back(delimiter);
-
-  // skip header
-  content.erase(0, content.find(delimiter) + 1);
-
-  // skip next two fields
-  content.erase(0, content.find(separator) + 1);
-  content.erase(0, content.find(separator) + 1);
+std::shared_ptr<Table> CsvParser::_create_table_from_meta() {
```

Comment:
This should `DebugAssert` that meta exists.

Reply:
If I'm not mistaken there is no way that meta does *not* exist at this point... Either it was loaded/passed successfully, or an exception was thrown in the process